ladybug_comfort/map/utci.py

```diff
@@ -0,0 +1,118 @@
+# coding=utf-8
+"""Utility functions for calculating Universal Thermal Climate Index (UTCI).
+
+This module is devoted to calculating UTCI with NumPy.
+"""
+import numpy as np
+from ..utci import _utci_polynomial_function
+
+
+def universal_thermal_climate_index_np(ta, tr, vel, rh):
+    """Calculate Universal Thermal Climate Index (UTCI) using a polynomial approximation.
+
+    This function here is a Python version of the original UTCI_approx
+    application written in Fortran. Version a 0.002, October 2009
+    The original Fortran code can be found at www.utci.org.
+
+    Note:
+        [1] Peter Bröde, Dusan Fiala, Krzysztof Blazejczyk, Yoram Epstein,
+        Ingvar Holmér, Gerd Jendritzky, Bernhard Kampmann, Mark Richards,
+        Hannu Rintamäki, Avraham Shitzer, George Havenith. 2009.
+        Calculating UTCI Equivalent Temperature. In: JW Castellani & TL
+        Endrusick, eds. Proceedings of the 13th International Conference
+        on Environmental Ergonomics, USARIEM, Natick, MA.
```

Comment:
I don't think we need to have the whole docstring copy/pasted here. Just say that this fuction is meant to do the esact same thing as the base `universal_thermal_climate_index` but it works with the Numpy data types.